### DISCUSSION TITLE
Import/Export Miitopia Miis

### DISCUSSION THREAD

**Discussion by sneedfeedseed** [Genwald/MiiPort#7] [Ideas/Feature Suggestions]
Hello love your program, thank you for making it! is it possible to implement a feature to able to import/export Miis made on Miitopia?

**Comment by Genwald** [comment]
I think that that would make for a good homebrew, but I don't think it fits into this one. This program's focus is specifically the normal Mii's that are shared between games. I might look into Miitopia for a different program when I've got more free time though.

**Comment by sneedfeedseed** [comment]
I understand, thank you for the response! I’ll be looking forward to your next works. :)

**Comment by Genwald** [comment]
@sneedfeedseed Wrote a quick python script for this today. It is a command line tool, but I think usage should be self explanatory if you've used one before. Use the `-h` flag to see help text.
https://gist.github.com/Genwald/ceffb596440e3952de6f9b1eb12e2428
